walk/git.go

```diff
@@ -25,12 +80,7 @@ func (g gitWalker) Root() string {
 	return g.root
 }
 
-func (g gitWalker) relPath(path string) (string, error) {
-	// quick optimization for the majority of use cases
```

Comment:
Removed this as I feel it wasn't adding much and was just a source of potentially more bugs.